linebot/get_follower_ids.go

```diff
@@ -33,6 +34,7 @@ type GetFollowerIDsCall struct {
 	ctx context.Context
 
 	continuationToken string
+	limit             string
```

Comment:
Please let me know why we use a string for "limit" not an integer.

According to the spec, it should be the number https://developers.line.biz/en/reference/messaging-api/#get-follower-ids-query-parameters

Reply:
Hi @kkdai ! thx for the question.

> Please let me know why we use a string for "limit" not an integer.

it depends on where to convert into string (since it is encoded into a query string),
and i thought letting the `bindLimit` method to do so is more clear to me.

however, having another function like `bindToQuery()` can be better idea. what do you think?
```
// GetFollowerIDsCall type
type GetFollowerIDsCall struct {
...
	continuationToken string
	limit             uint16
}

// Do method
func (call *GetFollowerIDsCall) Do() (*UserIDsResponse, error) {
	q := call.bindToQuery()
        res, err := call.c.get(call.ctx, call.c.endpointBase, APIEndpointGetFollowerIDs, q)
...
//
func (call *GetFollowerIDsCall) bindToQuery() url.Values {
	q := make(url.Values)
	if call.continuationToken != "" {
		q.Set("start", call.continuationToken)
	}
	if call.limit != 0 {
		q.Set("limit", strconv.FormatUint(uint64(call.limit), 10))
	}
	return q
}
```

Reply:
@kkdai As i suggested above, changed to let query parameters setting in `bindToQuery`, so `limit` is number until it's binded to the query. ( [c512e5e](https://github.com/line/line-bot-sdk-go/pull/363/commits/c512e5ee7833f40442f486ce468d8e1b3c0adedc) )

Reply:
Thanks, LGTM.